fibonacci search added (issue #7)

chaturvediabhay24 opened the pull request.

geekquad commented:
Hello @chaturvediabhay24 
Please go through the <a href="https://github.com/geekquad/AlgoBook/blob/master/CONTRIBUTING.md"> contributing</a> guidelines. The codes we want must be written in Python and not C++.
Please create an issue if you wish to add code in a language that we don't currently have here.

chaturvediabhay24 commented:
Hey @geekquad i have updated code in python

chaturvediabhay24 commented:
added bucker sort and fibbonacci search

chaturvediabhay24 commented:
Is there any problem with my code? I have changed my code in python also updated the two other files. If still there are more things needed to improve before merging PR please do let me know.

MAJOR-BEAST reviewed (changes requested):
You have some issue in the 346th line "execution_count": null
Just go through it once again.

MAJOR-BEAST reviewed (changes requested):
Code is good to go, but when you copy it from somewhere, at least so it a bit smartly. You also copied the line who actually contributed it.

MAJOR-BEAST reviewed (changes requested):
This code is also good to go, just make the required changes as in who wrote the code
